.sync.yml

```diff
@@ -1,4 +1,8 @@
 ---
+.gitattributes:
+  include:
+    '*.erb': eol=crlf
+    '*.epp': eol=crlf
 .gitlab-ci.yml:
   delete: true
 .travis.yml:
```

Comment:
Will you compare this one to https://github.com/ploperations/ploperations-webhook_proxy/blob/master/.sync.yml and see if there is more that should be done?